clients/AlphabotClientLibrary/AlphabotClientLibrary/AlphabotClientLibrary.Shared/Responses/ToggleResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AlphabotClientLibrary.Shared.Contracts;
+using AlphabotClientLibrary.Shared.Models;
```

Comment:
Remove unnecessary using directives.
```suggestion
using AlphabotClientLibrary.Shared.Contracts;
```